build/openresty/patches/ngx_lua-0.10.26_04-head-request-smuggling.patch

```diff
@@ -0,0 +1,27 @@
+From e5248aa8203d3e0075822a577c1cdd19f5f1f831 Mon Sep 17 00:00:00 2001
+From: lijunlong <lijunlong@openresty.com>
+Date: Sat, 9 Mar 2024 12:30:14 +0800
+Subject: [PATCH] bugfix: fixed HTTP HEAD request smuggling issue.
+
+---
+ src/ngx_http_lua_util.c |  6 ++++
+ t/020-subrequest.t      | 80 +++++++++++++++++++++++++++++++++++++++++
```

Comment:
Sorry for the late comment. It seems the test modification is not included in our patches. Is it because we don't run any tests from openresty?

Reply:
Yeah, we do not run any openresty test cases in our repo.